pkg/api/v20170701/types.go

```diff
@@ -115,8 +115,7 @@ const (
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
 	OrchestratorType    string `json:"orchestratorType" validate:"required"`
-	OrchestratorRelease string `json:"orchestratorRelease"`
```

Comment:
KubeConfig for loading values based on version (in addition to release) needs to be enabled: https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/const.go#L94.